entity/Comment.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Comment;
+
+use User\User;
+use DateTime;
+
+class Comment
+{
+    private User $user;
+    private string $comment;
+    private DateTime $createdAt;
+
+    public function __construct(User $user, string $comment)
```

Comment:
Можно поля класса определить сразу в конструкторе
```suggestion
    public function __construct(private User $user, private string $comment)
    {
    }
```